scripts/DocParser.js

```diff
@@ -294,18 +295,41 @@ function generateIndex () {
 				const filename = result.fullPath;
 				const json = jsonfile.readFileSync(filename);
 				try {
-					index.addDoc(jsonata(expression).evaluate(json));
+					const doc = jsonata(expression).evaluate(json);
+					doc.id = `${doc.title}|docs/modules/${doc.title}`;
```

Comment:
Can you share/document why we need to use this encoding here and in the block below?

Reply:
:yougotitbuddy: done.